src/ray/core_worker/core_worker.h

```diff
@@ -759,14 +759,14 @@ class CoreWorker : public rpc::CoreWorkerServiceHandler {
   void GetAsync(const ObjectID &object_id, SetResultCallback success_callback,
                 SetResultCallback fallback_callback, void *python_future);
 
-  /// Connect to plasma store for async futures
-  using PlasmaSubscriptionCallback = std::function<void(ray::ObjectID, int64_t, int64_t)>;
-
-  /// Set callback when an item is added to the plasma store.
+  /// Perform async get from in-memory store.
```

Comment:
this also does retrying?

Reply:
What do you mean? in terms of fetching from Plasma?